### PR TITLE
dwmblocks: init at 1.0

### PR DESCRIPTION
**Pull request by sophrosyne97**
###### Motivation for this change
As a dwm user I find it convenient to use dwmblocks as a simple status bar to display system information.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/dwmblocks/default.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchFromGitHub, libX11, patches ? [ ], writeText, conf ? null }:
+
+stdenv.mkDerivation {
+  name = "dwmblocks";
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/dwmblocks/default.nix:4]:
```suggestion
  pname = "dwmblocks";
  version = "unstable-XXXX-XX-XX";
```

please replace `XXXX-XX-XX` with the date of the commit.

File: pkgs/applications/misc/dwmblocks/default.nix [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  src = fetchFromGitHub {
+    owner = "torrinfail";
+    repo = "dwmblocks";
+    rev = "96cbb453e5373c05372fd4bf3faacfa53e409067";
+    sha256 = "00lxfxsrvhm60zzqlcwdv7xkqzya69mgpi2mr3ivzbc8s9h8nwqx";
+  };
+
+  nativeBuildInputs = [ libX11 ];
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/dwmblocks/default.nix:13]:
```suggestion
  buildInputs = [ libX11 ];
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by sophrosyne97** [comment]
Wow I am sorry I am still relatively new to working with github. I tried to use git rebase -i HEAD~5 with fixup to squash my last two commits but I was dumb and didn't realize that it would add in those other commits above. is there a simple way for me to clean up those other commits so it's just only the two that were requested to change for the pull request? Again sorry for the inconvenience

**Comment by SuperSandro2000** [comment]
I think the easiest way is ``git rebase @~17 -i`` and drop all the extra.

**Comment by sophrosyne97** [comment]
Thanks for the help that seems to have done the trick. The changes should be reflected in the commit in question now.

**Pull request merged**